src/index.js

```diff
@@ -1,37 +1,103 @@
+ * @param {*} bills array of bills exe: [{name:1,city:'Roma',type:'ambo',numbers:'1,4,66,76,32,90'},{...}]
+ * @returns {String} nice formatted string
+ */
+const checkBillsWinner = (extraction, bills) => {
+
+    if(bills === undefined){return  `Please restart the software and select the number of bills you want to play.`}
+
+    const arrayOfResultString = [] // array of string to list the number and city played that won and loose, as an infomration output
+    const winningBills = [] // array of string to print only the bills that won
+    const winningPrizes = [] // array of string ('x.xxx,xx €') of all the winning prize from winning bills 
+    const bets = [] // array of string ('x.xxx,xx €') to have a list of all the bets
+
+    bills.forEach(bill => {
+        bets.push(bill.bet)
+        if (bill.checkWinBill(extraction)){
+            arrayOfResultString.push(`Bill number ${bill.name} in the city of ${bill.city}  WON!`)
+            winningBills.push(bill.print(maxBillNChar,maxCityChar,maxTypeChar,maxNumbersChar,maxBetChar))
+            winningPrizes.push(bill.calculatePrize(grossPrizeTable))
+        }
+        else arrayOfResultString.push(`Bill number ${bill.name} in the city of ${bill.city} LOOSE!`)
+    })
+
+    //some caluclation for gross, net revenue and profit, changing the format between num and string currency for calculation and rappresantion
+    const winningPrizedFormattedNum = arrayCurrencyToNumber(winningPrizes) //array of num
+    const betsFormattedNum = arrayCurrencyToNumber(bets) // array of num
+
+    let totalGrossRevenue = winningPrizedFormattedNum.reduce((accumulator,currentValue) => accumulator + currentValue,0) // sum of array element output = num
+    const totalNetRevenue = calculateNetPrize(totalGrossRevenue) // output = string currency
+
+    totalGrossRevenue = numberToCurrency(totalGrossRevenue) // output = string currency
+    const totalNetRevenueFormatteNum = currencyToNumber(totalNetRevenue)  // output = num
+
+    let totalNetProfit = totalNetRevenueFormatteNum - (betsFormattedNum.reduce((accumulator,currentValue) => accumulator + currentValue,0))
+    totalNetProfit = numberToCurrency(totalNetProfit) // output = string currency
+
+    if(winningBills.length > 0) {
+        return`\n${printTotalExtraction(extraction)}\n${arrayOfResultString.join('\n')}\n\nAll bills played:\n${printBills(bills)}
+        \n\nWinning bills:\n${header}\n${winningBills.join('\n')}\n${footer}
+        \n\nTotal gross revenue = ${totalGrossRevenue}\nTotal net revenue = ${totalNetRevenue}\nTotal net profit = ${totalNetProfit}`}
+        return`\n${printTotalExtraction(extraction)}\n${arrayOfResultString.join('\n')}\nAll bills played:\n${printBills(bills)}
+        \nNo winning bills!`
+}
+
 
 //personal tests 
-console.log(printBills(bills))
\ No newline at end of file
+console.log(checkBillsWinner(allCityExtraction,bills))
```

Comment:
Why did you put this file (`index.js`) in the `src` directory? 
Shouldn't it be in the `root` directory?